### PR TITLE
FR-15720 - Remove entitlements V1 usage

### PR DESCRIPTION
**Pull request by doregg**
As a first step for removing V1 in rest API and admin box (js)

### CODE REVIEW

**Review by eran-frontegg — APPROVED** [review]

File: packages/vue/src/auth/entitlements.ts [hunk 1]
```diff
@@ -59,8 +46,8 @@ const useEntitlementsQueryData = (customAttributes?: CustomAttributes) => {
 */
 export const useFeatureEntitlements = (key: string, customAttributes?: CustomAttributes): ComputedRef<Entitlement> => {
   return computed(() => {
-    const { entitlements, attributes, isV2 } = useEntitlementsQueryData(customAttributes);
-    return getFeatureEntitlements(entitlements, key, attributes, isV2)
+    const { entitlements, attributes } = useEntitlementsQueryData(customAttributes);
+    return getFeatureEntitlements(entitlements, key, attributes, true)
```
Comment (yuvalotem1) [inline packages/vue/src/auth/entitlements.ts:50]: dont forget to remove isV2 from `getFeatureEntitlements`

File: packages/vue/src/auth/entitlements.ts [hunk 2]
```diff
@@ -72,8 +59,8 @@ export const useFeatureEntitlements = (key: string, customAttributes?: CustomAtt
 */    
 export const usePermissionEntitlements = (key: string, customAttributes?: CustomAttributes): ComputedRef<Entitlement> => {
   return computed(() => {
-    const { entitlements, attributes, isV2 } = useEntitlementsQueryData(customAttributes);
-    return getPermissionEntitlements(entitlements, key, attributes, isV2)
+    const { entitlements, attributes } = useEntitlementsQueryData(customAttributes);
+    return getPermissionEntitlements(entitlements, key, attributes, true)
```
Comment (yuvalotem1) [inline packages/vue/src/auth/entitlements.ts:63]: dont forget to remove `isV2` from `getPermissionEntitlements `

File: packages/vue/src/auth/entitlements.ts [hunk 3]
```diff
@@ -85,7 +72,7 @@ export const usePermissionEntitlements = (key: string, customAttributes?: Custom
 */
 export const useEntitlements = (entitledToOptions: EntitledToOptions, customAttributes?: CustomAttributes): ComputedRef<Entitlement> => {
   return computed(() => {
-    const { entitlements, attributes, isV2 } = useEntitlementsQueryData(customAttributes);
-    return getEntitlements(entitlements as any, entitledToOptions, attributes, isV2)
+    const { entitlements, attributes } = useEntitlementsQueryData(customAttributes);
+    return getEntitlements(entitlements, entitledToOptions, attributes, true)
```
Comment (yuvalotem1) [inline packages/vue/src/auth/entitlements.ts:76]: dont forget to remove `isV2` from `getEntitlements `

**Review by yuvalotem1 — APPROVED** [review]

File: packages/vue/src/auth/entitlements.ts (hunk 1 above)
Comment (doregg) [reply packages/vue/src/auth/entitlements.ts:50]: will take more cycles to do it.. it's like a chicken and egg problem

**Pull request merged**